lib/vty.h

```diff
@@ -182,6 +182,11 @@ struct vty_arg
 /* Small macro to determine newline is newline only or linefeed needed. */
 #define VTYNL  ((vty->type == VTY_TERM) ? "\r\n" : "\n")
 
+/* for compatibility */
+#define VTY_NEWLINE                               VTYNL
```

Comment:
What is the point of this? Nobody is using these macros in the code base.

Just on the off chance that you have some patches you need to fix up:

`$ find . -type f -name '*.[ch]' | xargs sed -i -e "s/VTY_NEWLINE/VTYNL/g"`